addons/l10n_fr_facturx_chorus_pro/models/account_edi_xml_ubl_bis3.py

```diff
@@ -0,0 +1,39 @@
+from odoo import models
+
+
+CHORUS_PRO_PEPPOL_ID = "0009:11000201100044"
+
+
+class AccountEdiXmlUBLBIS3(models.AbstractModel):
+    _name = "account.edi.xml.ubl_bis3"
+    _inherit = 'account.edi.xml.ubl_bis3'
+
+    """ See Pagero documentation: https://www.pagero.com/onboarding/aife/aife-en#requirements """
+
+    def _get_partner_party_identification_vals_list(self, partner):
+        """
+        Pagero doc states that the 'siret' of the final customer (that has the Chorus peppol ID) should be located in
+        the PartyIdentificiation node
+        """
+        vals = super()._get_partner_party_identification_vals_list(partner)
+        if (
+            partner.peppol_eas
+            and partner.peppol_endpoint
+            and partner.peppol_eas + ":" + partner.peppol_endpoint == CHORUS_PRO_PEPPOL_ID
+            and 'siret' in partner._fields
+            and partner.siret
+        ):
+            vals.append({
+                'id': partner.siret,
+            })
```

Comment:
```suggestion
            return [{
                'id': partner.siret,
            }]
```
Just to make sure `id` is unique?